### DISCUSSION TITLE
Finding the right key to scrap the data & Extracting the right information

### DISCUSSION THREAD

**Discussion by pow200492** [teamalgoritma/community#363] [Q&A] [answered]
need help to solve this issue
i already run through this code on mya jupyter notebook:
.......
import requests
url_get = requests.get('https://www.coingecko.com/en/coins/ethereum/historical_data/usd?start_date=2020-01-01&end_date=2021-06-30#panel')

from bs4 import BeautifulSoup 
soup = BeautifulSoup(url_get.content,"html.parser")
.......
and all show a good sign, but then when i tried to find the key using .find() and/or .find_all() it always give respond either
**<class 'NoneType'> or <class 'bs4.element.ResultSet'>** (i already tried all of the div, table, tr, even tbody and get the same result)
due to those issue whenever i want to check using print(xx.prettify()[1:500]) the reult keep saying AttributeError as follow;

**
![Screen Shot 2022-09-30 at 09 54 55](https://user-images.githubusercontent.com/111487102/193188072-0eab8571-e24c-432b-a988-85ac6b32189c.png)

![Screen Shot 2022-09-30 at 09 53 29](https://user-images.githubusercontent.com/111487102/193188068-9038dd10-a830-4e0f-85ef-6f25495fb864.png)

![Screen Shot 2022-09-30 at 09 34 14](https://user-images.githubusercontent.com/111487102/193188045-386c27f6-8b35-4bf4-842b-07f13a91a91c.png)**

Did I missed something here?

thank you

**Comment by hwulanayu** [comment, marked as answer]
The answer refers to this discussion: https://github.com/teamalgoritma/community/discussions/198
